src/web/core/src/main/java/org/geoserver/web/admin/GlobalSettingsPage.html

```diff
@@ -85,16 +85,16 @@ <h3><span><wicket:message key="internalSettings">Internal Settings</wicket:messa
         <label for="requestLogging"><wicket:message key="requestLogging">Enable Request Logging</wicket:message></label>
           <ul wicket:id="requestLoggingSub">
             <li class="choiceItem">
-              <input id="requestLogging" type="checkbox" wicket:id="requestLoggingBodies" />
-              <label for="requestLogging"><wicket:message key="requestLoggingBodies">Log Request Bodies</wicket:message></label>
```

Comment:
These were  generated with the same id, so the request logging label would check update several checkboxes all together!